src/util/config.cpp

```diff
@@ -776,8 +776,8 @@ configt::bv_encodingt parse_object_bits_encoding(
   const std::string &argument,
   const std::size_t pointer_width)
 {
-  const unsigned int object_bits = unsafe_string2unsigned(argument);
-  if(object_bits <= 0 || object_bits >= pointer_width)
```

Comment:
:pick: Would be better if the case where parsing fails and the `object_bits` are out of range were separated, also it would be helpful for debugging to put `argument` in the error message (this makes it easier to find if for example you've messed up a command line when copy&pasting, e.g. `--object-bits --function hello world 8`)

Reply:
Done.